src/print.rs

```diff
@@ -114,13 +115,13 @@ pub fn get_prompt(context: Context) -> String {
 
     let module_strings = root_module.ansi_strings_for_shell(context.shell, Some(context.width));
     if config.add_newline && context.target != Target::Continuation {
-        // continuation prompts normally do not include newlines, but they can
+        // continuation profiles normally do not include newlines, but they can
         writeln!(buf).unwrap();
     }
     write!(buf, "{}", ANSIStrings(&module_strings)).unwrap();
 
     if context.target == Target::Right {
-        // right prompts generally do not allow newlines
+        // right profiles generally do not allow newlines
```

Comment:
Same here: in almost all shells, if you look at the init scripts, the variable/function that controls this is named some variant of `RIGHT_PROMPT`.

Reply:
Bad replace; sorry